demo.py

```diff
@@ -74,9 +73,7 @@ def appapi_users(aapi):
     json_result = aapi.user_detail(275527)
     print(json_result)
     user = json_result.user
-    print(
-        "{}(@{}) region={}".format(user.name, user.account, json_result.profile.region)
-    )
+    print("{}(@{}) region={}".format(user.name, user.account, json_result.profile.region))
```

Comment:
Btw, should we rather introduce f-strings instead of `.format()`/%-formatting?

Reply:
Yes! this was only a reformat tho. We should do this in another PR.